src/components/CategoryProgress.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledCategoryProgress = styled.div`
+  font-size: 32px;
+  margin: 10px;
+  .page {
+    display: inline-block;
+  }
+  .CategoryProgressAmount {
+    color: #3ddc97;
+  }
+  .CategoryProgressBudget {
+    color: #9b9b9b;
+  }
+  .CategoryProgressMeter {
+    height: 1px;
```

Comment:
a few adjustments:
1. set the height to `4px`
2. border-radius will only work up to half of the height, so `2px` is the max that will have any effect
3. no need for padding, the guide you copies from had a slightly different styling need than us
4. adding `overflow: hidden` on the parent wrapper will also 'mimic' the border radius on the child, so move it to here.
5. you might not even need position absolute, the child should align just fine to the parent.